seacatauth/batman/elk.py

```diff
@@ -32,8 +33,7 @@ class ELKIntegration(asab.config.Configurable):
 		"password": "",
```

Comment:
`asab.tls.SSLContextBuilder` apparently also uses config option called `password` so this could cause unexpected behavior. Not sure how to solve this. You can rename these batman options to `elastic_username` and `elastic_password`. That would be a breaking change though.